tests/sentry/api/id_slug_path_params/test_doc_integration_slug.py

```diff
@@ -0,0 +1,26 @@
+from unittest.mock import patch
+
+from django.test import TestCase
+
+from sentry.testutils.cases import BaseTestCase
+from sentry.testutils.helpers.options import override_options
+from sentry.testutils.silo import no_silo_test
+
+from .test_id_or_slug_path_params_mixin import APIIdOrSlugTestMixin
+
+
+@no_silo_test
+class DocIntegrationSlugTests(BaseTestCase, TestCase, APIIdOrSlugTestMixin):
+    databases: set[str] | str = "__all__"
```

Comment:
I think you can inherit from the sentry [`TestCase`](https://github.com/getsentry/sentry/blob/d8dd1ab445de4c594ae6beb1f67788ece03658f0/src/sentry/testutils/cases.py#L489) here instead of the two testcases, which also let's you get rid of the `databases` part